src/styles/typography/index.md.njk

```diff
@@ -99,6 +99,27 @@ Use the `govuk-link--no-visited-state` modifier class where it is not helpful to
 
 {{ example({group: "styles", item: "typography", example: "link-no-visited-state", html: true, open: true}) }}
 
+If it's an external link to a non-government website, make that clear in the link text. For example, 'read advice on writing link text from [name of organisation]'. There's no need to say explicitly that you're linking to an external site. [Do not use an external link icon](https://designnotes.blog.gov.uk/2016/11/28/removing-the-external-link-icon-from-gov-uk/).
+
+### Opening links in a new tab
+
+Avoid opening links in a new tab or window. It can be disorienting - and [can cause accessibility problems for people who aren't able to perceive the link visually](https://www.w3.org/TR/WCAG20-TECHS/G200.html).
+
+If you need a link to open in a new tab - for example, to stop the user losing information they’ve entered into a form - then include the words ‘opens in new tab’ as part of the link.
```

Comment:
My service includes several links in the footer. Because we don't want users to lose their place in the service, these open in new tabs with a close button. Each link uses hidden text to say `(opens in a new window)`.

Example:
![Screenshot 2020-03-10 at 11 25 21](https://user-images.githubusercontent.com/2204224/76308079-f2cdb880-62c1-11ea-9010-8517079ec723.png)

Is the recommendation to include the text visually?

If so, would this be the recommendation?:
![Screenshot 2020-03-10 at 11 27 31](https://user-images.githubusercontent.com/2204224/76308188-26104780-62c2-11ea-9146-408cf04e9777.png)

Reply:
can you expand a bit on 'we don't want users to lose their place in the service' - would pressing 'back' work?

Reply:
@edwardhorsford how about something like this?

> If you're displaying lots of links together and want to save space and avoid repetition, consider applying `span class="govuk-visually-hidden"` to the 'opens in new window' text. This will mean the text is visually hidden but still accessible to screen readers.

cc @hannalaakso @joelanman

Reply:
Sounds good @StephenGill 👍 I think we need to add a code example though, the explanation of code in the text is not clear. It should be
```
<a href="#" class="govuk-link" rel="noreferrer noopener" target="_blank">govuk-link <span class="govuk-visually-hidden">(opens in new tab)</span></a>
```

Reply:
@StephenGill I think that's better. I agree in general it's good to be explicit - but it's practically hard for my use case - which I think will be common for many services' footers.

Reply:
Telling people that a link opens in a new tab is not just for screen reader users, though. So, visually hiding that information is no use to people with cognitive impairments.

Another way to solve this is to add something like "(all links open in a new tab)" to the heading of that link section. Although it would be great to test this with users.

Reply:
If 'opens in a new tab' message wasn't semantically part of the link mark up, wouldn't that mean that screen reader users are likely to miss it?

Reply:
I wouldn't say it's likely but certainly possible.

Navigating by heading is the most popular way to navigate content with a screen reader. (According to all the WebAIM surveys, e.g. https://webaim.org/projects/screenreadersurvey8/#finding). So it's likely a screen reader user will already have heard the heading before encountering the link.

It all depends on how they navigate the page. If they navigate by links or tab through, then it's more likely they'll miss it.
But someone who doesn't use a screen reader might equally miss it.

Making sure the link purpose is clear just from the link text is a WCAG AAA requirement (2.4.9). But according to the AA requirement (2.4.4) it's fine if it is clear within context. And a heading gives enough context.

Reply:
you could do both hidden text and a heading

Reply:
Yes, was thinking about something like

```
If you're displaying lots of links together and want to save space and avoid repetition, consider doing both of the following:

  - adding a line of text before the links saying 'the following links open in a new tab'
  - applying `span class="govuk-visually-hidden"` to the 'opens in new window' part of the link text, so that part of the link text is visually hidden but still accessible to screen readers

```
Does that look ok?

Reply:
My interpretation of what his might look like in practice:

<img width="904" alt="Screenshot 2020-03-18 at 11 11 25" src="https://user-images.githubusercontent.com/2204224/76954983-40bd6e80-6909-11ea-99b3-6dd058d023e6.png">

Reply:
That sounds good to me @StephenGill. :+1:

Reply:
is 'These links open in a new tab:' simpler?

Reply:
@StephenGill It might be worth an example if this is going in the design system?

As I see it the guidance is to directly label where you can (inline links mostly), or for things like footers, prepend it.

I also suspect this might need extra css for the footer component to acheive.

Reply:
We've now added the text by @StephenGill in the guidance in this PR.

We've also got a card to make it possible to add 'The following links open in a new tab' text with the footer component: https://github.com/alphagov/govuk-frontend/issues/1782.